src/SfxWeb/src/app/views/partition/backups/backups.component.ts

```diff
@@ -41,14 +52,88 @@ export class BackupsComponent extends PartitionBaseControllerDirective {
       new ListColumnSetting('raw.LsnOfLastBackupRecord', 'Lsn of last Backup Record'),
       new ListColumnSetting('raw.CreationTimeUtc', 'Creation Time UTC'),
     ]);
-    console.log(this);
+    this.debouncerHandlerSubscription = this.debounceHandler
+    .pipe(debounceTime(400), distinctUntilChanged())
```

Comment:
whats the benefit for using this? Is there a network call involved when filtering the list?

Reply:
Thanks Jeffrey, I had made these changes by referring to event store component. I have made the necessary changes in the code.